Zach_Page/dojo_dashboardII/app/controllers/dojos_controller.rb

```diff
@@ -0,0 +1,27 @@
+class DojosController < ApplicationController
+  def index
+    @dojos = Dojo.all
+  end
+
+  def new
+
+  end
+
+  def create
+    dojo = Dojo.new(dojo_params)
+    if dojo.valid?
```

Comment:
I'd recommend doing dojo.save as it returns true or false if it can. Validations run as a part of the save call.